client/src/components/Forms/User/SignUpForm.js

```diff
@@ -38,6 +38,15 @@ function SignUpForm() {
 						name="Password"
 					/>
 
+                    <input
```

Comment:
Hey, indentation here isn't really consistent

Reply:
Hye,
i think whole the form input even this one got the same indentation 

![image](https://user-images.githubusercontent.com/45979612/153090060-0705ccd4-f682-47b7-99d7-623d5a4ee531.png)